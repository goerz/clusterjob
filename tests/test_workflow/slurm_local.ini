[Attributes]
backend = slurm
shell = /bin/bash
rootdir = ~/jobs
workdir = job1

[Resources]

queue = exec
nodes = 1
threads = 1
mem = 20
