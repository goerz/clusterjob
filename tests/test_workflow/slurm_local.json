[
  {
    "args": [
      [
        "mkdir",
        "-p",
        "/home/clusterjob_test/jobs/job1"
      ],
      null
    ],
    "kwargs": {
      "ignore_exit_code": false,
      "ssh": "ssh"
    },
    "response": ""
  },
  {
    "args": [
      [
        "sbatch",
        "test_clj.slr"
      ],
      null,
      "~/jobs",
      "job1"
    ],
    "kwargs": {
      "ignore_exit_code": true,
      "ssh": "ssh"
    },
    "response": "Submitted batch job 5460095\n"
  },
  {
    "args": [
      [
        "squeue",
        "-h",
        "-o",
        "%T",
        "-j",
        "5460095"
      ],
      null
    ],
    "kwargs": {
      "ignore_exit_code": true,
      "ssh": "ssh"
    },
    "response": " RUNNING\n"
  },
  {
    "args": [
      [
        "scancel",
        "5460095"
      ],
      null
    ],
    "kwargs": {
      "ignore_exit_code": true,
      "ssh": "ssh"
    },
    "response": ""
  },
  {
    "args": [
      [
        "mkdir",
        "-p",
        "/home/clusterjob_test/jobs/job1"
      ],
      null
    ],
    "kwargs": {
      "ignore_exit_code": false,
      "ssh": "ssh"
    },
    "response": ""
  },
  {
    "args": [
      [
        "sbatch",
        "test_clj.slr"
      ],
      null,
      "~/jobs",
      "job1"
    ],
    "kwargs": {
      "ignore_exit_code": true,
      "ssh": "ssh"
    },
    "response": "Submitted batch job 5460096\n"
  },
  {
    "args": [
      [
        "squeue",
        "-h",
        "-o",
        "%T",
        "-j",
        "5460096"
      ],
      null
    ],
    "kwargs": {
      "ignore_exit_code": true,
      "ssh": "ssh"
    },
    "response": " RUNNING\n"
  },
  {
    "args": [
      [
        "squeue",
        "-h",
        "-o",
        "%T",
        "-j",
        "5460096"
      ],
      null
    ],
    "kwargs": {
      "ignore_exit_code": true,
      "ssh": "ssh"
    },
    "response": ""
  },
  {
    "args": [
      [
        "sacct",
        "--format=state",
        "-n",
        "-j",
        "5460096"
      ],
      null
    ],
    "kwargs": {
      "ignore_exit_code": true,
      "ssh": "ssh"
    },
    "response": " COMPLETED \n COMPLETED \n"
  }
]
