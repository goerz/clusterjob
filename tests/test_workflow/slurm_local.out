slurmd[its-cs325]: Unable to get current working directory: No such file or directory
####################################################
Job id         : 5460096
Job name       : test_clj
Workdir        : /gpfs/home07/ag-koch/storage/uk006169/jobs/job1
Submission Host: its-cs1.its.uni-kassel.de
Compute Node   : its-cs325
Job started on its-cs325.its.uni-kassel.de Thu Feb 25 22:44:29 CET 2016
Current directory: /gpfs/home07/ag-koch/storage/uk006169/jobs/job1
####################################################
Job Finished:  Thu Feb 25 22:47:29 CET 2016
